raisinbread/instruments/EEG_Acquisition_form.linst

```diff
@@ -0,0 +1,95 @@
+textarea{@}bad_channels{@}Register all bad channels
+header{@}{@}<h6>For net placement question:<br/> <b>Excellent</b> - meets all criteria for proper net placement.<br/> <b>Average</b> - any 1-2cm shift from proper placement (1 pinky finger width distance).<br/> <b>Poor</b> - any >2 cm shift from proper placement (>1 pinky finger width distance). </h6>
+select{@}net_placement{@}Net placement{@}'na'=>'n/a'{-}'excellent'=>'Excellent'{-}'average'=>'Average'{-}'poor'=>'Poor'
+textarea{@}net_placement_comment{@}Any comment on net placement
+select{@}net_placement_photos_taken{@}Placement photos taken?{@}'yes'=>'Yes'{-}'no'=>'No'
+select{@}acquisition_software{@}Acquisition software used{@}'eemagine'=>'Eemagine'{-}'neuro_scan'=>'(Neuro)scan'{-}'insep'=>'InsEP'{-}'biosemi'=>'BioSemi'{-}'netstation'=>'NetStation'{-}'brain_vision_recorder'=>'Brain Vision Recorder'{-}'emotiv'=>'Emotiv'{-}'actiview605_lores'=>'ActiView605-Lores'{-}'eeprobe'=>'EEProbe'{-}'tmsi'=>'TMSi'{-}'other'=>'Other'
+text{@}acquisition_software_other{@}If other, please specify
+select{@}recording_reference{@}Recording reference used{@}'left_mastoid'=>'left mastoid'{-}'right_mastoid'=>'right mastoid'{-}'left_earlobe'=>'left earlobe'{-}'right_earlobe'=>'right earlobe'{-}'physically_linked_mastoids'=>'physically linked mastoids'{-}'physically_linked_earlobes'=>'physically linked earlobes'{-}'cz'=>'Cz'{-}'tip_nose'=>'tip of the nose'{-}'balanced_non_cephalic_reference'=>'balanced non-cephalic reference'{-}'other'=>'Other'
+text{@}recording_reference_other{@}If other, please specify
+text{@}number_active_channels{@}Number of active channels
+select{@}number_active_channels_status{@}Number of active channels{@}NULL=>''{-}'not_answered'=>'Not Answered'
+select{@}eog_channels{@}EOG channels{@}'no_eog_channels'=>'no EOG channels'{-}'two_heog_channels_only'=>'two HEOG channels only'{-}'two_veog_channels_only'=>'two VEOG channels only'{-}'two_heog_two_veog_channels'=>'two HEOG and two VEOG channels'{-}'other'=>'Other'
+text{@}eog_channels_other{@}If other, please specify
+textarea{@}artefacted_channels{@}Artefacted channels
+header{@}{@}<b><h3>EEG Acquisition</h3></b>
+select{@}bad_electrodes_present{@}Were there any bad electrodes during impedances?{@}'yes'=>'Yes'{-}'no'=>'No'
+textarea{@}bad_electrodes{@}If Yes, please register all bad electrodes
+textarea{@}hardware_filters_applied{@}What hardware filters were applied before acquisition?
+textarea{@}software_filters_applied{@}What software filters were applied during acquisition?
+time{@}eeg_session_start_time{@}EEG session Start time
+select{@}eeg_session_start_time_status{@}EEG session Start time{@}NULL=>''{-}'dnk'=>'DNK'{-}'refusal'=>'Refusal'{-}'not_answered'=>'Not Answered'
+time{@}eeg_session_end_time{@}EEG session End time
+select{@}eeg_session_end_time_status{@}EEG session End time{@}NULL=>''{-}'dnk'=>'DNK'{-}'refusal'=>'Refusal'{-}'not_answered'=>'Not Answered'
```

Comment:
does refusal make sense for an end time field?

Reply:
Good question, it was generated through the parser. Looking at the first iteration, it was this line: 
`$this->addHourMinElement("eeg_session_start_time", "EEG session Start time");`
but the code automatically generated two grouped fields (`eeg_session_start_time` and `eeg_session_start_time_status`). The `eeg_session_start_time_status` field has these values.
The code is here: https://github.com/aces/Loris/blob/main/php/libraries/NDB_BVL_Instrument.class.inc#L1741
I will remove both `*_status` fields for this case.